assets/app/view/user.rb

```diff
@@ -89,9 +91,17 @@ def reset_settings
       input_elm(:bg2).value = default_for(:bg2)
       input_elm(:font2).value = default_for(:font2)
       input_elm(:red_logo).checked = false
+
       TILE_COLORS.each do |color, hex_color|
         input_elm(color).value = hex_color
       end
+
+      ROUTE_COLORS.each_with_index do |hex_color, index|
```

Comment:
this looks copy pasted

Reply:
?

Reply:
how is this different from the top part that looks the same? in lib/settings

Reply:
this resets the inputs to defaults